### PR TITLE
DEV-19527 Adding documentation for custom properties: Array

### PR DESCRIPTION
**Pull request by egollas**
## Changes
- Adds documentation for custom properties: Array

## JIRA Story
https://jira.inmoment.com/browse/DEV-19527

## How to see it
- Go to http://localhost:4567/api#responses, you should see sample properties with `_array` suffix
- Go to http://localhost:4567/javascript#custom-properties, you should see a sample property with `_array` suffix, and the "Setting up Custom Properties" sections should include the "List" custom property documentation as well.

### CODE REVIEW

File: source/includes/_custom_fields.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ window.wootricSettings = {
     pricing_plan:'Enterprise', // TODO: The current user's pricing plan.
     total_purchase_amount: 12, // Integer representing the user's total purchases with the key suffixed with "_amount"
     last_order_date: 1350466020 // Integer representing the date (Unix timestamp format) of the user's last order with the key suffixed with "_date"
+    items_array: ['ITEM1', 'ITEM2'] // List of values representing the possible items with the key suffixed with "_array"
```
Comment (diegoserranoa) [inline source/includes/_custom_fields.md]: This shouldn't be an array, it's a comma separated string.

File: source/includes/_responses.md [hunk 2]
```diff
@@ -52,7 +52,8 @@ curl -H "Authorization: Bearer myaccesstoken" "https://api.wootric.com/v1/respon
                 "pricing_plan": "Small Business",
                 "product_plan": "Mobile",
                 "purchase_date": 1473177881,
-                "revenue_amount": 5000
+                "revenue_amount": 5000,
+                "items_array": ["ITEM1", "ITEM2"]
```
Comment (diegoserranoa) [inline source/includes/_responses.md]: Same here...

File: source/includes/_responses.md [hunk 3]
```diff
@@ -76,7 +77,8 @@ curl -H "Authorization: Bearer myaccesstoken" "https://api.wootric.com/v1/respon
                 "pricing_plan": "Small Business",
                 "product_plan": "Web",
                 "purchase_date": 1473177888,
-                "revenue_amount": 5000
+                "revenue_amount": 5000,
+                "items_array": ["ITEM1", "ITEM2"]
```
Comment (diegoserranoa) [inline source/includes/_responses.md]: Same as above.

**Review by diegoserranoa — CHANGES_REQUESTED** [review]
The values for `_array` properties should be strings. Example: `'Item1, Item2'`

I think they *could* arrays but the web-beacon hasn't been update to handle that yet., so better to stick with comma separated strings.

File: source/includes/_custom_fields.md (hunk 1 above)
Comment (jessekathryn) [reply source/includes/_custom_fields.md]: OK, got it, thanks Diego

Comment (jessekathryn) [reply source/includes/_custom_fields.md]: > The values for `_array` properties should be strings. Example: `'Item1, Item2'`
> 
> I think they _could_ arrays but the web-beacon hasn't been update to handle that yet., so better to stick with comma separated strings.

Can we discuss when you return?

**Comment by jessekathryn** [comment]
Screen Shot 2022-03-24 at 12 40 50 PM
Screen Shot 2022-03-24 at 12 42 07 PM

File: source/includes/_custom_fields.md (hunk 1 above)
Comment (jessekathryn) [reply source/includes/_custom_fields.md]: ok updated!

**Review by diegoserranoa — APPROVED** [review]

**Pull request merged**